### ISSUE TITLE
Consider creating a  FACS module 

### ISSUE THREAD

**Issue by pnejad** [HumanCellAtlas/metadata-schema#619]
**For which schema is a change/update being suggested?**

enrichment_protocol.json

**What should the change/update be?**

New FACS module that could be included in enrichment protocols. This module could include fields such as:
- nozzle_size
- sort_mode
- analyte
- analyte_detector
- analyte_reporter
- clone 
- instrument_model
- instrument_manufacturer 
- more ...?

**What new field(s) need to be changed/added?**

TBD

We should consider [MIFlowCyt](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2773297/) (The Minimum Information about a Flow Cytometry Experiment) standards.

 
**Why is the change requested?**

Our current enrichment protocol does not capture details of FACS enrichment well.

[NIHMS76346-supplement-Supp_Rec.pdf](https://github.com/HumanCellAtlas/metadata-schema/files/2591107/NIHMS76346-supplement-Supp_Rec.pdf)

**Comment by malloryfreeberg** [comment]
@pnejad thanks for the ticket.

Just wanted to note that New Issues aren't assigned to anyone. Issues are assigned to people when they are In Progress. Please review the Zenhub policies in the [Runbook](https://github.com/HumanCellAtlas/metadata-schema/blob/master/docs/Zenhub_runbook.md#new-issues) .

**Comment by zperova** [comment]
I agree that we need to improve the metadata we are collecting on FACS.
I had a look through the MIFlowCyt 1.0 and I would argue that collecting all the mentioned components is better suited in protocols.io rather than as part of HCA metadata. HCA metadata should be relevant from the viewpoint of a data consumer, such as what the user might search for in the data portal that will be helpful to guide their analysis/future experiments. We need to reflect the details needed to understand the experimental design but not the gritty details of how it has been done. Where we draw this line is a good question that requires discussion. I think this ticket needs a bit of research on what is the information regarding FACS that HCA can afford taking in.

**Comment by malloryfreeberg** [comment]
Based on the Metadata call on Dec 3, we need to write up some use cases from a computational biologist perspective of what metadata they would want and/or need from a FACS experiment to do their work.

**Comment by jishuxu** [comment]
just got sometime to look into this. it would be nice if we can "standardize" meta data for FACS or any kind of cell sorter. but in the reality, it is very very difficult. Last week, i just got a chance to sit next to an experimental scientist to observe how she was sorting cells. From what I observed, the algorithm used by sorters might be different from machine to machine. when you get to gating or selecting population step, there are a lot of individual judgement involved. 
We can start to think about the meta data for FACS, there is my list I think would be useful.
1. manufacture, such as machine type, analysis software
2. anti-bodies used
3. sort cells into tube or plate
4. some of machine can provide final analysis files(*,csv) and we might request them if possible.

**Comment by lauraclarke** [comment]
Thanks for this @jishuxu 

Can you given us insight into how this might be used downstream in computational analysis as that will help us decide how best to represent it in the schema

**Comment by malloryfreeberg** [comment]
This issue has been brought up on the last 2 metadata data calls.

Currently waiting for data consumer use cases (from @pnejad and @matthewspeir) to outline what a computational scientist might need to know about a FACS protocol. This issue will be brought up on the next metadata call.

**Comment by pnejad** [comment]
@malloryfreeberg  This is what I asked the ux team (cBeta slack channel) on 12/18:

> The metadata team has a few topics that we’d like feedback on. Could some of our questions be incorporated into cbeta interviews?
> 
> We are interested in the following:  1) How to view primary tissue, cell lines, organoids (tabs? Filters? search?) 2) How to capture “organ model” for cell lines 3) Feedback from computational biologist on recording FACS metadata

@gabsie  You mentioned you could possibly incorporate some of these questions in January interviews. Is that still the case? 

It would be really helpful to get feedback from people like Jishu (comment above).

**Comment by pnejad** [comment]
I spoke to 2 people from the Weissman lab (postdoc and Ph.D. student) and got the same answer from both. They would not trust FACS data unless the **FACS plots** and **gating strategies** are provided. The postdoc also said that it should be a "best practice" to provide post-sort analysis with FACS data, but mentioned that can't be provided if sorting in single-cell mode.
